packages/tfchain_client/src/tft_price.ts

```diff
@@ -11,6 +11,46 @@ class QueryTFTPrice {
     const res = await this.client.api.query.tftPriceModule.tftPrice();
     return res.toPrimitive() as number;
   }
+
+  @checkConnection
+  async getTFTBillingRateMUSD(): Promise<number> {

```

Comment:
The uppercase M sounds like it's millions not milli
```suggestion
  async getTFTBillingRatemUSD(): Promise<number> {

```

Reply:
The capital 'M' in MUSD was used strictly to adhere to camel case convention. Within this repo, we exclusively deal with standard USD and milli-USD; we do not use the Mega prefix anywhere. Therefore, the limited scope of our units ensures there is no ambiguity for the reader.